fe1-web/src/features/lao/components/LaoItem.tsx

```diff
@@ -50,13 +51,12 @@ const LaoItem = ({ lao, isFirstItem, isLastItem }: IPropTypes) => {
         getNetworkManager().connect(address),
       );
 
-      const channel = getLaoChannel(lao.id.valueOf());
```

Comment:
I can't comment a few lines above but I have to mention that `onPress` isn't a great name for a `onPress` callback.
The ListItem attribute already says that it's called on press, the function name should say what it actually does :-)

Reply:
That is true :)